target/linux/ramips/mt7620/base-files/etc/board.d/02_network

```diff
@@ -301,6 +306,11 @@ ramips_setup_macs()
 		[ -n "$lan_mac" ] || lan_mac=$(cat /sys/class/net/eth0/address)
 		wan_mac=$(macaddr_add "$lan_mac" 1)
 		;;
+	hootoo,tm05)
+		lan_mac=$(mtd_get_mac_binary factory 0x4)
```

Comment:
You don't need this, as you set ethernet to the same address in DTS.

Reply:
OK, wasn't sure - it did seem redundant, but figured it was safe ... :-). Will remove it though. Thanks!